source/_integrations/synologydsm.markdown

```diff
@@ -50,6 +50,10 @@ password:
   description: The password of the user to connect to the Synology NAS.
   required: true
   type: string
+api_version:
+  description: Define DSM version to allow backward compatibility with 5.x.
```

Comment:
Can you specity that the `api_version` can be only 5 or 6 ?

As written in your library PR "Added reverse compatibility with DSM 5.2" ProtoThis/python-synology#9 --> files --> README.rst

Reply:
Sorry, I did change the documentation, not sure if I did it properly ?